src/main/openapi/ds-discover-openapi_v1.yaml

```diff
@@ -1031,3 +1082,30 @@ components:
           format: int32
         message:
           type: string
+          
+          
+ # TODO: Inject this from properties when DRA-139 has been implemented
+  securitySchemes:
+
+    KBOAuth:
+      type: oauth2
+      description: 'This API uses OAuth 2 with the implicit grant flow. [More info](https://api.example.com/docs/auth)'
+      flows:
+        implicit:   # <---- OAuth flow(authorizationCode, implicit, password or clientCredentials)
+          authorizationUrl: 'https://keycloak-devel-01.kb.dk/realms/ds-poc/protocol/openid-connect/auth'
```

Comment:
I think this URL should be defined in the config file and referenced here as: '${config:openapi.the.key.in.yaml}'

Reply:
Injected now. Added property to devel-yaml. And placeholder for behavior-yaml